pkg/virt-controller/watch/drain/disruptionbudget/disruptionbudget.go

```diff
@@ -406,10 +406,18 @@ func (c *DisruptionBudgetController) execute(key string) error {
 	}
 
 	for i := range pdbs {
+		if vmi.IsFinal() {
```

Comment:
@dasionov Lets also check that ` vmi.DeletionTimestamp == nil` because when VMI deletion is ongoing the PDB will be deleted